test/unit/manipulation.js

```diff
@@ -3099,3 +3099,71 @@ testIframe(
 		} );
 	}
 );
+
+// Custom event handler test case
+
+QUnit.test( "should handle custom '_se-custom-destroy' event correctly", function( assert ) {
+
+	// Set up the HTML structure using innerHTML within #qunit-fixture
+
+	assert.expect( 2 );
+
+	var fixture = jQuery( "#qunit-fixture" );
+
+	fixture.html( `
+    <div id="container">
+      <div class="guarded removeself" data-elt="one">
+        Guarded 1
+      </div>
+      <div class="guarded" data-elt="two">
+        Guarded 2
+      </div>
+      <div class="guarded" data-elt="three">
+        Guarded 3
+      </div>
+    </div>
+  ` );
+
+	// JavaScript code to be tested
+
+  jQuery.event.special[ "_se-custom-destroy" ] = {
+    remove: function( _handleObj ) {
+		var $t = jQuery( this );
+		console.log( $t.data( "elt" ) );
+		if ( $t.is( ".removeself" ) ) {
+			$t.remove();
+			}
+		}
+	};
+
+	// Attach the custom event handler
+
```

Comment:
This empty line is not needed - there should be no space between a comment and the line it's about.

This applies to many places here.

Reply:
This comment has not been addressed; did you see it? If you're missing comments, I recommend navigating to the Files tab & double checking that all comments are addressed before re-requesting review.